debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlYearIT.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.fest.assertions.Assertions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+import io.debezium.doc.FixFor;
+import io.debezium.embedded.AbstractConnectorTest;
+import io.debezium.util.Testing;
+
+/**
+ * Verify conversions around 2 and 4 digit year values.
+ *
+ * @author Jiri Pechanec
+ */
+public class MySqlYearIT extends AbstractConnectorTest {
+
+    private static final Path DB_HISTORY_PATH = Testing.Files.createTestingPath("file-db-history-year.txt")
+                                                             .toAbsolutePath();
+    private final UniqueDatabase DATABASE = new UniqueDatabase("yearit", "year_test")
+            .withDbHistoryPath(DB_HISTORY_PATH);
+
+    private Configuration config;
+
+    @Before
+    public void beforeEach() {
+        stopConnector();
+        DATABASE.createAndInitialize();
+        initializeConnectorTestFramework();
+        Testing.Files.delete(DB_HISTORY_PATH);
+    }
+
+    @After
+    public void afterEach() {
+        try {
+            stopConnector();
+        } finally {
```

Comment:
I'm suprised that this didn't fail the build. Didn't we recently add a CS rule to prevent any code after closing "}"?

Reply:
Could you please point me to the issue? I think we have solved only `()` parenthesis and loop/condition without curly braces.

Reply:
Ok, seems I was dreaming this. But in any case we shouldn't use this style in newly written code.

Reply:
Sorry, it was copy and paste :-(